app/scripts/controllers/reviewRegistration.js

```diff
@@ -123,7 +123,7 @@ angular.module('confRegistrationWebApp')
       }
     };
 
-    var postPayment = function(currentPayment){
+    var postPayment = function(currentPayment){ // jshint ignore:line
```

Comment:
It is existing code that someone else created.
JSHint:  ‘postPayment’ was used before it was defined.
I think it is safe to ignore due to JavaScript hoisting.